framework/src/media/audio/audio_manager.h

```diff
@@ -342,32 +374,64 @@ unsigned int get_input_bytes_to_frame(unsigned int bytes);
 unsigned int get_output_frame_count(void);
 
 /****************************************************************************
- * Name: get_output_frames_to_byte
+ * Name: get_card_output_frames_to_byte
+ *
+ * Description:
+ *   Get the byte size of the given frame value with the channel value
+ *   supported by the card for output stream.
+ *
+ * Input parameter:
+ *   frames: the target of which byte size is returned.
+ *
+ * Return Value:
+ *   On success, the byte size of the frame in output stream. Otherwise, 0.
+ ****************************************************************************/
+unsigned int get_card_output_frames_to_byte(unsigned int frames);
```

Comment:
Change 
unsigned int get_card_output_frames_to_byte(unsigned int frames);
to
uint32_t get_card_output_frames_to_byte(uint32_t frames);

Reply:
Right. Thx!